src/Akka.Logger.NLog/NLogLogger.cs

```diff
@@ -40,9 +40,9 @@ private static void Log(LogEvent logEvent, Action<NLogger, LogEvent> logStatemen
         public NLogLogger()
         {
             Receive<Error>(m => Log(m, (logger, logEvent) => LogEvent(logger, NLogLevel.Error, m.LogSource, m.Cause, m.Message)));
```

Comment:
I understand why you would want to use logEvent.LogSource. Because you clean that up in parent call. But you forgot to change this for the Error scenario. Any specific reason ?

Reply:
LogEvent does not contain an exception property ?

Reply:
Because only the Error LogEvent has the Cause-Exception-Property, so the capture remains. So I just left it unchanged. But I can change it if you like?

Reply:
Yes please. I would rather have it be in line with the rest as much as possible.

Reply:
@Danthar Done